src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -160,6 +171,10 @@
   &.pf-m-plain {
     color: var(--pf-c-dropdown__toggle--m-plain--Color);
 
+    > i {
```

Comment:
We can't reliably count on an `<i>` element here. For example, they're `<svg>` in react. I think you could change this to `> *`...